.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "deployer/common"]
 	path = deployer/common
 	url = https://github.com/openshift/origin-integration-common
+[submodule "kibana-proxy"]
```

Comment:
For the purposes of the release scripts we can call it anything. That being the case I thought it might be nice to have it named according to the function it plays in this system. I don't feel strongly about it, though, if you think `openshift-auth-proxy` is better to minimize confusion I'll do that.